net/netcheck/netcheck.go

```diff
@@ -433,9 +433,23 @@ func makeProbePlan(dm *tailcfg.DERPMap, ifState *netmon.State, last *Report) (pl
 	had4 := len(last.RegionV4Latency) > 0
 	had6 := len(last.RegionV6Latency) > 0
 	hadBoth := have6if && had4 && had6
+	// #13969 ensure that the home region is always probed so that spurious
+	// non-incremental measures do not lead to a home region flip.
```

Comment:
what are "spurious non-incremental measures"? I'm having a hard time reading that. Can you reword somehow?